content/beginner/190_secrets/consume-a-secret.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Access the Secret from a Pod"
+date: 2019-04-09T00:00:00-03:00
+weight: 11
+draft: false
+---
+
+#### Deploy a Pod to Consume the Secret
+Create a YAML file (podconsumingsecret.yaml) with the following pod definition:
+
+```
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: consumesecret
+spec:
+  containers:
+  - name: shell
+    image: amazonlinux:2018.03
+    command:
+      - "bin/bash"
+      - "-c"
+      - "cat /tmp/test-creds && sleep 10000"
+    volumeMounts:
+      - name: sec
+        mountPath: "/tmp"
+        readOnly: true
+  volumes:
+  - name: sec
+    secret:
+      secretName: test-creds
+```
+
+Deploy the pod on your EKS cluster:
+```bash
+kubectl --namespace secretslab \
+        apply -f podconsumingsecret.yaml
+```
+Output: 
+{{< output >}}
+pod/consumesecret created
+{{< /output >}}
+
+Attach to the pod and attempt to access the secret:
+```
+kubectl --namespace secretslab exec -it consumesecret -- cat /tmp/test-creds
+```
+
+Output: 
+{{< output >}}
+am i safe?
+{{< /output >}}
+
+Let's see if the CloudTrail event for our secret retrieval is now visible. If you go to CloudTrail you should see a record available if you search for the Event type ```Decrypt``` with output similar to the following screenshot. If the event hasn't shown up yet, wait a few minutes and try again.
```

Comment:
Let's make "go to cloudtrail" a deep-link to the spot in the cloudtrail console we want them to land.

something like:
`https://console.aws.amazon.com/cloudtrail/home?events&#/events?EventName=Decrypt`

Reply:
I missed this one as well. I will add it.

Reply:
Added the deep-link. Thanks!